src/main/docs/guide/kubernetes-informer.adoc

```diff
@@ -0,0 +1,63 @@
+The `micronaut-kubernetes-informer` module integrates the https://javadoc.io/doc/io.kubernetes/client-java/latest/io/kubernetes/client/informer/SharedIndexInformer.html[SharedIndexInformer] that is part of official Kubernetes SDK and simplifies its creation. The Informer is similar to a Watch but an Informer tries to re-list and re-watch to hide failures from the caller and provides a store of all the current resources. Note that the default official implementation https://javadoc.io/doc/io.kubernetes/client-java/latest/io/kubernetes/client/informer/SharedInformerFactory.html[SharedInformerFactory] creates shared informers per the Kubernetes resource type. However the module `micronaut-kubernetes-informer` creates namespace scoped informers of the Kubernetes resource type, meaning that the informer is shared per specified namespace and kind.
+
+First you need add a dependency on the `micronaut-kubernetes-informer` module:
+
+dependency:io.micronaut.kubernetes:micronaut-kubernetes-informer:{version}[]
+
+Then create a bean that implements https://javadoc.io/doc/io.kubernetes/client-java/latest/io/kubernetes/client/informer/ResourceEventHandler.html[ResourceEventHandler] with the Kubernetes resource type of your choice and add the link:{api}/io/micronaut/kubernetes/client/informer/Informer.html[@Informer] annotation trough which you provide the configuration for the https://javadoc.io/doc/io.kubernetes/client-java/latest/io/kubernetes/client/informer/SharedIndexInformer.html[SharedIndexInformer].
```

Comment:
Just as a tip, you can use `ann:kubernetes.client.informer.Informer[]` to simplify link creation